deploy/clowdapp.yaml

```diff
@@ -51,6 +51,8 @@ objects:
           value: ${NEWER_RELEASEVER_REPOS}
         - name: NEWER_RELEASEVER_CSAF
           value: ${NEWER_RELEASEVER_CSAF}
+        - name: ENABLE_GO_IMPLEMENTATION
+          value: ${ENABLE_GO_IMPLEMENTATION}
```

Comment:
make separate env for every endpoint so we can enable them in prod gradually once they are tested, change the name to something like ENABLE_GO_CVES